### PR TITLE
Removes magicarps from gold slime pool

### PR DESCRIPTION
**Pull request by jughu**
You shouldnt be introducing magic into a non magic round.
Magicarps allows to the following.
Infinite minerals(door bolt)
Emagged stuff(bolt of change)
Instadeath shots(i ded)
Fireball(i also ded)
Teleport(ok.)
ASH DRAKES SYNDICATE BORGS ETC WHAT ISNT SUPPOSED T BE ON STATION REE

### CODE REVIEW

**Comment by Arianya** [comment]
You can get cult materials in a non-cult round, syndicate objects in non-traitor/nukie rounds, etc. A big part of SS13 is paranoia, and having cross-contamination is part of that.

From a balance PoV, the gold slime reaction is exceptionally capable of backfiring and requires a fair amount of investment (plus being at the mercy of RNG) to get, so I'm not opposed to it existing as it currently does.

File: code/modules/events/wizard/magicarp.dm [hunk 1]
```diff
@@ -37,6 +37,7 @@
 	projectilesound = 'sound/weapons/emitter.ogg'
 	maxHealth = 50
 	health = 50
+	gold_core_spawnable = 0
```
Comment (Cyberboss) [inline code/modules/events/wizard/magicarp.dm]: FALSE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/events/wizard/magicarp.dm (hunk 1 above)
Comment (jughu) [reply code/modules/events/wizard/magicarp.dm]: What false

Comment (CPTANT) [reply code/modules/events/wizard/magicarp.dm]: Bools are false, not 0

**Comment by kevinz000** [comment]
out of every xenobiology mobs magicarps are the one mob that will destabilize the round most of the time by just /existing/.

**Comment by InsaneHyena** [comment]
Fun removal

**Comment by Xhuis** [comment]
i ded

**Comment by Ratvar** [comment]
Golden slime reactions can "backfire" ? Uh, what year is it?

**Comment by jughu** [comment]
>backfire>inject with blood and be good for whole shift

**Comment by Durkel** [comment]
Polymorph magicarps have fucked up so many round that they're a huge issue. There's a reason syndieborgs and all other shit aren't available every round.

**Comment by Arianya** [comment]
Even blood reaction mobs can get hostile (some for a little as "being near them", which is pretty easy to do when they spawn right next to you.

And so @Ratvar stands in the shame corner for trying to name and shame people who regularly play the game :roll_eyes:

**Comment by Ratvar** [comment]
Stop spawning mos right next to you?

**Review by Cyberboss — APPROVED** [review]

**Comment by KorPhaeron** [comment]
Fun removal

**Comment by DoctoreMad** [comment]
>Instadeath shots(i ded)
Fireball(i also ded)

At least he has the decency to say it.

**Comment by SgtxRootbeer** [comment]
>adding stuff that shouldnt be in a round to a round
good, it breaks the meta. we all love breaking the meta, no?

**Comment by KingPhilipIII** [comment]
>Fun police.

**Comment by Robustin** [comment]
I hate magicarps with every fiber of my being. I've had more than one round absolutely ruined because some xenobiologist got a wand-of-death magicarp that racked up a double-digit body count. 

Yet, as is my theme lately, I support keeping it in because it's the kind of crazy shit that defines this game and while I was pissed getting one-shot by a fucking fish from the edge of my screen, it was equally entertaining watching the fish go on a rampage. 

Only thing that would be nice is shifting stuff around so that mining+science didn't have a monopoly on crazy OP stuff. Xenobio to medbay when.

**Comment by DoctoreMad** [comment]
>Xenobio to medbay

This idea is so crazy it might actually work.

**Comment by KorPhaeron** [comment]
I think most players would rather have rounds break down in entertaining ways rather than have us strive for perfect balance, especially when you are nearing the 40 minute mark

**Comment by jughu** [comment]
>entertaining

**Pull request closed without merging**